### PR TITLE
Improve navbot patrol

### PR DESCRIPTION
**Pull request by aUniqueUser**
- Fixes patrol getting stuck because it walks to area closest to localplayer causing it to walk between a couple nodes for a long time.
- Makes patrol stay nearby the enemy objective on payload and control points

### CODE REVIEW

File: src/hacks/NavBot.cpp [hunk 1]
```diff
@@ -944,6 +902,52 @@ bool captureObjectives()
...
 
+// Roam around map
+bool doRoam()
+{
+    static Timer roam_timer;
+    // Don't path constantly
+    if (!roam_timer.test_and_set(200))
+        return false;
+
+    // RED team, nav near enemy objective if possible
```
Comment (BenCat07) [inline src/hacks/NavBot.cpp:913]: Logical flaw, some gamemodes have blue capture aswell.

**Review by BenCat07 — CHANGES_REQUESTED** [review]

**Review by BenCat07 — APPROVED** [review]
Approval of owner or @TotallyNotElite also needed, but LGTM

**Review by BenCat07 — APPROVED** [review]
LGTM

**Review by TotallyNotElite — APPROVED** [review]
lgtm

**Pull request merged**